common/menu.c

```diff
@@ -271,6 +311,23 @@ void menuStartup() {
     //menuCreateMsgBox(780, 300, "This is a test");
 }
 
+void themeMenuStartup() {
+    if(hbmenu_state != HBMENU_DEFAULT) return;
+    hbmenu_state = HBMENU_THEME_MENU;
+    char tmp_path[PATH_MAX];
+
+    snprintf(tmp_path, sizeof(tmp_path)-1, "%s%s%s%s%s%s",DIRECTORY_SEPARATOR, "config", DIRECTORY_SEPARATOR, "nx-hbmenu" , DIRECTORY_SEPARATOR, "themes");
+
+    themeMenuScan(tmp_path);
+    if(globalPreset == THEME_PRESET_DARK)
+        theme_icon_small = downscaleImg(theme_icon_dark_bin, 256, 256, 140, 140, IMAGE_MODE_RGB24);
```

Comment:
Move this into menuStartup, it should only be initialized once (themeMenuStartup can be called multiple times).

Reply:
done